client-core/src/service/wallet_state_service.rs

```diff
@@ -60,8 +60,8 @@ where
         name: &str,
         passphrase: &SecUtf8,
```

Comment:
I suggest to add an extra option here and the underlying `wallet state` service to always return all UTXOs.